sound/soc/sof/ipc4-topology.h

```diff
@@ -72,6 +72,7 @@
  * @lp_mode: Low power mode
  * @mem_usage: Memory usage
  * @state: Pipeline state
+ * @use_chain_dma: flag to indicate if the firmware should use chained DMA
```

Comment:
shall use chained DMA. It's not a recommendation, it's a requirement. The topology would not work if the firmware ignored this information.